src/main/java/com/synopsys/integration/alert/channel/email/EmailChannel.java

```diff
@@ -98,8 +100,16 @@ public void distributeMessage(DistributionEvent event) throws IntegrationExcepti
 
     public void sendMessage(EmailProperties emailProperties, Set<String> emailAddresses, String subjectLine, String formatType, EmailAttachmentFormat attachmentFormat, MessageContentGroup messageContent) throws IntegrationException {
         String topicValue = null;
+        Optional<String> subTopicValue = Optional.empty();
         if (!messageContent.isEmpty()) {
             topicValue = messageContent.getCommonTopic().getValue();
+            subTopicValue = messageContent.getSubContent()
+                                .stream()
+                                .map(ProviderMessageContent::getSubTopic)
+                                .flatMap(Optional::stream)
+                                .filter(linkableItem -> MessageBuilderConstants.LABEL_PROJECT_VERSION_NAME.equals(linkableItem.getName()))
```

Comment:
I don't think we need to check this. We know that the sub-topic is the project-version for Black Duck and if there is ever another provider, it will be likely that we need to include this for feature parity.

Reply:
A license limit notification won't include a version unless I'm mistaken. The API of ProviderMessageContent doesn't make it clear that the sub topic is the version so given that API I suggested filtering to be sure you have the project version sub topic.  The initial code just had find first.  But with your suggestion to join the sub topics the filtering may not make sense anymore.  If we remove the filter portion of the stream then I would like to see a comment above that states we're assuming the sub-topic is the version.   Just in case we get a bug later on that the subject isn't correct we have a comment to refresh our memory on the assumption in case it comes into play.